test/functional/wallet_basic.py

```diff
@@ -47,9 +50,6 @@ def check_fee_amount(self, curr_balance, balance_with_fee, fee_per_byte, tx_size
     def get_vsize(self, txn):
         return self.nodes[0].decoderawtransaction(txn)['vsize']
 
-    def labels_value(self, name="", purpose="receive"):
```

Comment:
Would combine this commit with bb87d8aac71dc87d1f11c26bd97fa6dc7b650a75, as you are mostly touching LoC that you touched in the that one as well.